### ISSUE TITLE
Low SFX volume in Chocolate Heretic

### ISSUE THREAD

**Issue by Paar86** [chocolate-doom/chocolate-doom#971]
The volume of the sound effects is very low in comparison to vanilla Heretic. On the other hand, the volume of the music seems OK. Tested with beta 3.

**Comment by fragglet** [comment]
Are you using Windows, and what are you using for music playback?

Can you confirm the bug is still present in the released 3.0.0?

**Comment by Paar86** [comment]
Yeah, I'm using Windows 10. For music I use native MIDI playback.

**Comment by fragglet** [comment]
Thanks - can you try to 3.0.0 release and see if it fixes the problem? We recently merged something that will probably make a difference.

**Comment by Paar86** [comment]
I would like to add that the SFX volume is somehow dependant on the music volume (lowering the music volume lowers the volume of the sounds). Maybe it's related?

**Comment by chungy** [comment]
Can you verify you have `chocolate-midiproc.exe`?

**Comment by Paar86** [comment]
That solved the problem with midi volume changing SFX volume. Unfortunately, the SFX volume is still low compared to other source ports or vanilla game using Dosbox.

**Comment by PepsimanRUS** [comment]
fragglet, why is the problem not resolved in 2 years?
